StyleCop.Analyzers/StyleCop.Analyzers/ReadabilityRules/ReadabilityResources.resx

```diff
@@ -477,6 +477,18 @@
   <data name="SA1134Title" xml:space="preserve">
     <value>Attributes must not share line</value>
   </data>
+  <data name="SA1135CodeFix" xml:space="preserve">
+    <value>Qualify using directive</value>
+  </data>
+  <data name="SA1135Description" xml:space="preserve">
+    <value>All using directives must be qualified.</value>
+  </data>
+  <data name="SA1135MessageFormat" xml:space="preserve">
+    <value>The using directive '{0}' must be qualified to '{1}'</value>
```

Comment:
:question: How about the following?

> Using directive for namespace '{0}' must be qualified

:memo: In this message, `{0}` is the value you referred to as `{1}` in the original message.